.samples/websocket-chat-app.yaml

```diff
@@ -0,0 +1,10 @@
+displayName: WebSocket Chat Application
+description: Provides simultaneous two-way communication over a single TCP connection.
+componentType: service
+buildPack: node
+repositoryUrl: https://github.com/wso2/choreo-samples/
+componentPath: /websocket-chat-app/websocket-chat-service-nodejs
```

Comment:
Let's use `/websocket-chat-app` as the componentPath here.  Example - .samples/reading-list-app.yaml

Also let's add a readme to /websocket-chat-app to describe the contents of the subfolders. Example - reading-list-app/README.md